tensorflow/g3doc/resources/index.md

```diff
@@ -37,6 +37,8 @@ The TensorFlow community has created many great projects around TensorFlow, incl
 * [TensorFlow tutorials](https://github.com/pkmital/tensorflow_tutorials)
 * [Scikit Flow - Simplified Interface for TensorFlow](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/learn/python/learn)
 * [Caffe to TensorFlow model converter](https://github.com/ethereon/caffe-tensorflow)
+* [Bitfusion's` GPU-enabled AWS EC2 TensorFlow AMI](https://github.com/bitfusionio/amis/tree/master/awsmrkt-bfboost-ubuntu14-cuda75-tensorflow) ([Launch AMI](https://aws.amazon.com/marketplace/pp/B01EYKBEQ0))
+ * Comes with: Tensorflow 9, CuDNN 5, Cuda 7.5, Keras, Jupyter and Udacity deep learning courseware
```

Comment:
Is it possible instead to permanent link to your 'latest AMI', and then have that page list the information about the TF/CuDNN versions of that AMI?  Otherwise this information is going to become stale very quickly.